app/src/main/res/layout/fragment_setting.xml

```diff
@@ -4,27 +4,19 @@
   ~ License, v. 2.0. If a copy of the MPL was not distributed with this
   ~ file, You can obtain one at http://mozilla.org/MPL/2.0/.
   -->
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    tools:context=".view.SettingFragment">
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There's a lot of mixing between layouts here. Sometimes `RelativeLayout`, sometimes `ConstraintLayout`, sometimes `LinearLayout`.

Suggest we use `FrameLayout`s for layouts with one view, and `ConstraintLayout` everywhere else.

Reply:
With the linear layout, the view is nested inside so that the "settings" header and back button are viewable. If this changes to a frame layout or constraint layout, the header and back button no longer show up. 
(the rest of the .xml files are LinearLayouts)